src/style-spec/reference/v8.json

```
@@ -2524,6 +2500,72 @@
     "value": "*",
     "doc": "A filter selects specific features from a layer."
   },
+  "filter_symbol": {
+    "type": "boolean",
+    "doc": "Expression used to determine whether a symbol is being displayed or not. Symbols support `dynamic-filtering`, so this expression can account for the current state of the view by incorporating `[\"pitch\"]` and `[\"distance-from-center\"]` as part of the expression.",
```

I think this description means to _define_ `dynamic-filtering`, but I read it like it was referencing an elsewhere-defined concept I should know about. I modified the wording a little bit, but I think the biggest change was "meaning" so that it reads like a self-contained definition.

```suggestion
    "doc": "Expression which determines whether or not to display a symbol. Symbols support `dynamic-filtering`, meaning this property can use the `[\"pitch\"]` and `[\"distance-from-center\"]` expressions to reference the current state of the view.",
```